QuickFIXn/Session.cs

```diff
@@ -619,7 +649,7 @@ internal void Next(MessageBuilder msgBuilder)
             catch (TagException e)
             {
                 if (e.InnerException is not null)
-                    Log.OnEvent(e.InnerException.Message);
+                    Log.Log(LogLevel.Error, "{Message}", e.InnerException.Message);
```

Comment:
Is there a reason why to not include the exception for this one when its an error level so you would get a stack trace etc when you have the ones below including them

Reply:
one more thing: Such an exception doesn't actually indicate any error in the running app.  Sometimes a badly-implemented counterparty can send bad messages, and your app just has to deal.  And the correct way to deal is to 35=3 reject.